README.md

```diff
@@ -69,13 +69,11 @@ class MyApplication(object):
 
     def __init__(self, *args, **kwargs):
         boto = Boto(
-            logger=self.logger,
-            stats=self.stats,
+            access_key='my-access-key',
+            access_key='my-secret-key',
```

Comment:
Are you re-assigning access_key, should these be different variable names, or it creating an array?